src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -236,16 +236,37 @@
         </Grid.ColumnDefinitions>
 
         <!-- row 0: -->
-        <!-- deprecation indicator -->
+
+        <!-- warnings indicator -->
         <imaging:CrispImage
-                      x:Name="_deprecationIndicator"
+                      x:Name="_warningIndicator"
                       Grid.Row="0"
                       Grid.Column="0"
-                      Visibility="{Binding IsPackageDeprecated,Converter={StaticResource BooleanToVisibilityConverter}}"
+                      Visibility="{Binding IsPackageWithWarnings,Converter={StaticResource BooleanToVisibilityConverter}}"
                       VerticalAlignment="Center"
-                      ToolTip="{x:Static nuget:Resources.Label_Deprecated}"
                       AutomationProperties.Name="{x:Static nuget:Resources.Label_Deprecated}"
-                      Moniker="{x:Static catalog:KnownMonikers.StatusWarning}"/>
+                      Moniker="{x:Static catalog:KnownMonikers.StatusWarning}">
+          <imaging:CrispImage.ToolTip>
+            <StackPanel>
+              <TextBlock
+                Margin="0, 0, 0, 8"
+                Visibility="{Binding IsPackageVulnerable,Converter={StaticResource BooleanToVisibilityConverter}}">
+                <TextBlock.Text>
+                  <MultiBinding Converter="{StaticResource StringFormatConverter}">
```

Comment:
Does `StringFormatConverter` exists in WPF ? I'm not sure if we are using Windows Community Toolikit inside VS. 

https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.converters.stringformatconverter?view=win-comm-toolkit-dotnet-7.0

Reply:
We seem to have our own implementation of a StringFormatConverter which is used in the PackageItemViewModel and I am using here as well.